RecoHGCal/TICL/plugins/TrackstersMergeProducer.cc

```diff
@@ -27,6 +27,7 @@ class TrackstersMergeProducer : public edm::stream::EDProducer<edm::GlobalCache<
   void produce(edm::Event &, const edm::EventSetup &) override;
   static void fillDescriptions(edm::ConfigurationDescriptions &descriptions);
   void energyRegressionAndID(const std::vector<reco::CaloCluster> &layerClusters, std::vector<Trackster> &result) const;
+  void computeTime(std::vector<TICLCandidate> &resultCandidates);
```

Comment:
This could be made `private`, isn't it?
(I think also `energyRegressionAndID` can be made `private`)

Reply:
By the way, wouldn't a name like "assignTimeToCandidates", or something similar, be more appropriate for this method?

Reply:
Done in the latest commit.